### DISCUSSION TITLE
How to use a link on desktop to start msys2 build of txs or how to make it portable

### DISCUSSION THREAD

**Discussion by octaeder** [texstudio-org/texstudio#2464] [Q&A] [answered]
I start txs in msys2 with ./texstudio.exe. If I create a link on that file on my desktop, it fails. Is this possible at all? I know that links work with portable versions. So maybe I need to know how to make my build portable. Please give some advice.

Thx

**Comment by sunderme** [comment, marked as answer]
probaly because the dlls are not found. txs portable has all dlls in the same directory.
